packer/plugin-getter/plugins_test.go

```diff
@@ -27,6 +27,28 @@ var (
 	pluginFolderWrongChecksums = filepath.Join("testdata", "wrong_checksums")
 )
 
+func TestChecksumFileEntry_init(t *testing.T) {
+	expectedVersion := "v0.3.0"
+	req := &Requirement{
+		Identifier: &addrs.Plugin{
+			Hostname:  "github.com/ddelnano/packer-plugin-xenserver",
+			Namespace: "",
+			Type:      "xenserver",
+		},
```

Comment:
From how i understand the [plugin source syntax](https://www.packer.io/docs/plugins#source-addresses) of packer
packer doesn't use the normal url syntax, and instead constructs the plugin name like this:

https://$hostname/$namespace/packer-plugin-$type

it should be like this:
```
Identifier: &addrs.Plugin{
	Hostname:  "github.com",
	Namespace: "ddelnano",
	Type:      "xenserver",
},
```

I have run the tests like with this change and they do pass.

Reply:
I meant to circle back and make sure I got these fields right but I forgot. Updated this in my latest revision.